build/build-dt-report-resources.js

```diff
@@ -23,8 +23,20 @@ const htmlReportAssets = require('../lighthouse-core/report/html/html-report-ass
  */
 function convertToAsciiAndWriteFile(name, content) {
   assert(content);
+
+  /** @type {string} */
+  let prefix;
+  if (name.endsWith('.html')) {
+    // This will not support unicode characters in inline stylesheets or js.
```

Comment:
There actually isn't any non ascii characters in the HTML files (afaik). This is just for (semi) completeness.

How would you detect nonascii in a style rule (and not elsewhere in the html)? I'm ok with just deferring this until there is nonascii in these files

Reply:
> How would you detect nonascii in a style rule (and not elsewhere in the html)?

seems like it doesn't matter since we don't have any right now? It does seem better to just throw and be explicit about it

Reply:
I can throw on any ascii. Cannot allow ascii in html but throw on ascii in style/script (w/o actually parsing). I'll do the former.

Reply:
> I can throw on any ascii.

that's what I was at least trying to say :)